### PR TITLE
py-wandb: add v0.16.6

### PR DESCRIPTION
**Pull request by thomas-bouvier**

### CODE REVIEW

**Comment by thomas-bouvier** [comment]
Seeing `Invalid dependency type: rim`, I have no idea what it means

File: var/spack/repos/builtin/packages/py-wandb/package.py [hunk 1]
```diff
@@ -29,9 +30,11 @@ class PyWandb(PythonPackage):
     depends_on("py-typing-extensions", type=("build", "run"), when="^python@:3.9")
 
     depends_on("py-pyyaml", type=("build", "run"))
-    depends_on("py-click@7:", type=("build", "run"))
+    depends_on("py-click@7:", type=("build", "run"), when="@0.13")
+    depends_on("py-click@7.1:", type=("build", "rim"), when="@0.16:")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-wandb/package.py]: rim -> run

Comment (thomas-bouvier) [reply var/spack/repos/builtin/packages/py-wandb/package.py]: 😖😖😖

**Comment by thomas-bouvier** [comment]
Ready to be merged @adamjstewart

**Comment by adamjstewart** [comment]
I'm no longer maintaining this package, maybe someone else can review.

**Comment by thomas-bouvier** [comment]
Maybe @tldahlgren can help? :)

**Review by wdconinc — COMMENTED** [review]
`py-pathtools` is not needed anymore. New version of `py-click` as of 0.15.5 per https://github.com/wandb/wandb/commit/5973f4436ddc7b6c218cee7fb1eb4cf04b2547bd.

**Review by wdconinc — APPROVED** [review]
Checksum verified.

**Pull request merged**